### ISSUE TITLE
Pass Language group tab info to profiles to store

### ISSUE THREAD

**Issue by RobinaSanderson** [AtlasOfLivingAustralia/seasonal-calendar#57]
If the web service pushing information to the associated profiles collection does not yet include Language tab information this issue  will cover  transferring that data.

**Comment by m-hope** [comment]
With the move to "Option 2".  This issue needs to examine all the fields defined above and store them appropriately as follows using the mapping table at the bottom of the SC confluence page (https://confluence.csiro.au/display/ALASD/Seasonal+calendars+-+user+requirements+June+2018):
- if the field already exists at a collection level in Profiles, then the information is to be passed to Profiles.
- if the field has a relevance to all collections (see mapping table), but doesn't exist, the field should be created in Profiles and then passed to Profiles as above.
- if the field has no relevance to Profiles collections it should be stored locally.

Note: the work for this issue should be a part of the SC data services issue #73.

**Comment by RobinaSanderson** [comment]
Renaming this to just deal with storing the language group information, not necessarily in Profiles.

**Comment by RobinaSanderson** [comment]
Hi @patkyn - I think I'm finished with this. Let me know if I've missed anything and you have any questions.

**Comment by patkyn** [comment]
Hi @RobinaSanderson, should the map details(long, lat, zoom) be stored in Opus in Profiles or in seasonal calendars database?

**Comment by RobinaSanderson** [comment]
Hi @patkyn  and @sbearcsiro  - If there is a better option technically I'd go with that as the map area is relevant to both the seasonal calendar and the supporting profile collection. i.e. the IEK information in both the seasonal calendar and the collection is relevant to the area in the map. I don't know if the map is relevant to other collections, I think this would only be the case with collections covering areas less than state wide like the Olkola or the Yugul Mangi Rangers collections but if the rest of the language group info is in the seasonal calendar we could store it there. Do you have a preference?

**Comment by RobinaSanderson** [comment]
@patkyn and I had a discussion about whether language group information should be stored in the same table as the seasonal calendar or whether it should have it's own table. Currently, language group is only used with a seasonal calendar and only one language group is relevant to each seasonal calendar so the information could be stored together.
It is possible that in the future an indigenous group will want to record language group information without having a seasonal calendar, for example the Olkola group and the Yugul Mangi Rangers might want to store information about their language group without having an associated seasonal calendar, and we thought if we store the language group information in a separate table it might be easier to pull out of seasonal calendars at a later point, without having to spend a lot of effort now, future proofing for something we may never do.
@m-hope  and @sbearcsiro  does this seem reasonable to you, or should we just go easiest route  now and worry about the future if it comes up later?

**Comment by sbearcsiro** [comment]
As it's 1-1 right now and a calendar always has a language group and a language group always has a calendar then IMHO we just store it in the same table *unless* it's definitely going to be split in the near future.  The difference for extract information later is simply `select * from languages` vs `select <language fields> from calendars` vs the additional overhead of managing a 1-1 relationship (which tbf isn't that much but I feel its slightly more onerous than selecting a list of fields).

**Comment by patkyn** [comment]
Yes, lets just do the simple solution for now combining Language Group fields into Calendar
